### PR TITLE
Created a sign-sbom script to build cyclonedx library and sign the sbom

### PR DESCRIPTION
**Pull request by julian55455**
Created a sign-sbom script to build cyclonedx library and sign the sbom

### CODE REVIEW

**Comment by karianna** [comment]
@julian55455 Linter fails.

**Comment by andrew-m-leonard** [comment]
@julian55455 This will be the jenkins called script, use https://github.com/adoptium/temurin-build/blob/master/build-farm/sign-releases.sh
as an example. so it will be called with the sbom JSON artifact to be signed copied from the upstream job to a folder called workspace/target/...json

eg:
```
workspace/target/OpenJDK17U-sbom_arm_linux_hotspot_17.0.6_10.json
```

You sign it, and then the Jenkins job itself will archive the signed artifact.

File: build-farm/sign-sbom.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (andrew-m-leonard) [inline build-farm/sign-sbom.sh:1]: Use this example as the basis of what this script needs to do: https://github.com/adoptium/temurin-build/blob/master/build-farm/sign-releases.sh

**Review by andrew-m-leonard — CHANGES_REQUESTED** [review]

**Comment by zdtsw** [comment]
this script wont do any harm, as long as it is not get called.
but good to have #3211 fixed first then correct the script and add a caller

**Pull request closed without merging**